bin/check-ec2-cpu_balance.rb

```diff
@@ -101,7 +107,7 @@ def run
     level = 0
     instances.reservations.each do |reservation|
       reservation.instances.each do |instance|
-        next unless instance.instance_type.start_with? 't2.'
+        next unless instance.instance_type.start_with?(*config[:instance_families].split(','))
```

Comment:
I think `*` is a typo? Also it might make sense to do this when passed in the var, for example: https://github.com/sensu-plugins/sensu-plugins-aws/blob/16.1.0/bin/check-cloudwatch-alarms.rb#L53 this makes it easier to use in other locations in this check if that makes sense.

Reply:
> I think * is a typo? 

No, it's the splat operator because `start_with` does not take an array as argument but as many strings as we want.

> Also it might make sense to do this when passed in the var, for example: https://github.com/sensu-plugins/sensu-plugins-aws/blob/16.1.0/bin/check-cloudwatch-alarms.rb#L53 this makes it easier to use in other locations in this check if that makes sense.

Totally, I was not aware of this `proc` attribute, I'll do that !

Reply:
oh this might be a splat operator just saw the error with CI and assumed it was a typo.

Reply:
Ah thanks for the explanation sorry I didnt see it, was not refreshed when I was looking.